packages/appkit-utils/src/ethers/EthersHelpersUtil.ts

```diff
@@ -41,10 +41,14 @@ export const EthersHelpersUtil = {
       params: [
         {
           chainId: EthersHelpersUtil.numberToHexString(caipNetwork.id),
-          rpcUrls: caipNetwork.rpcUrls.default.http,
+          rpcUrls: [caipNetwork.rpcUrls.default.http[0]],
           chainName: caipNetwork.name,
-          nativeCurrency: caipNetwork.nativeCurrency,
-          blockExplorerUrls: caipNetwork.blockExplorers,
+          nativeCurrency: {
```

Comment:
Are these happy with the Viem's type definitions?

Reply:
I didn't get why the previous version not working fine 🤔 
```js
rpcUrls: caipNetwork.rpcUrls.default.http
```

Reply:
I got this error : https://stackoverflow.com/questions/42376464/uncaught-domexception-failed-to-execute-postmessage-on-window-an-object-co